dashboard/app/assets/stylesheets/application.scss

```diff
@@ -484,16 +484,20 @@ img.video_thumbnail {
 
 .main {
   padding: 10px;
-  a:link, a:visited, a:hover, a:active {
-    @include main-font-bold;
-    color: $brand_secondary_default;
-    background-color: transparent;
-    text-decoration: underline;
-  }
-  a:hover {
-    color: $brand_secondary_dark;
-    text-decoration: underline;
+
+  &:not(:has(.visual-block)) {
```

Comment:
Hopefully we don't have to make many more exceptions but I worry a bit about having to update this whenever we run into this issue again...though I don't know of any better way to do this, so this makes sense!